src/lib/components/questions/question-core.svelte.ts

```
@@ -82,6 +80,10 @@ export function createQuestionCardSession(opts: QuestionCardSessionOpts) {
 	);
 	const showWarmingState = $derived(isPoolWarming);
 	const showErrorState = $derived(!isLoading && questionLoadFailed && !isPoolWarming);
+	const optionMarks = $derived<OptionMarks>({
+		struckOptionIds: new Set(struckOptionIds),
```

ESLint

[error] Found a mutable instance of the built-in Set class. Use SvelteSet instead.

(svelte/prefer-svelte-reactivity)